src/VisualStudio/Core/Def/LanguageService/AbstractLanguageService`2.cs

```diff
@@ -138,7 +137,6 @@ protected virtual void GetServices()
         this.EditorOptionsService = this.Package.ComponentModel.GetService<EditorOptionsService>();
         this.Workspace = this.Package.ComponentModel.GetService<VisualStudioWorkspaceImpl>();
         this.EditorAdaptersFactoryService = this.Package.ComponentModel.GetService<IVsEditorAdaptersFactoryService>();
-        this.AnalyzerFileWatcherService = this.Package.ComponentModel.GetService<AnalyzerFileWatcherService>();
```

Comment:
we would create the instance her.e  HOwever:

1. nothing ever read this feild in this type. 
2. the constructor for that type does nothing.

Reply:
i noticed this as i wanted to move these calls into the constructor of AbstractLangService, and i wanted to make sure none of them were doing anything wonky.